test/test_backends/test_s3.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, name):
         self.name = name
         self.keys = {}
 
-    def get_key(self, key):
+    def get_key(self, key, validate=True):
```

Comment:
Why do you set the default to `True` here? Is this to mimic the unmocked behavior?